src/module-rebuilder.ts

```diff
@@ -119,6 +120,23 @@ export class ModuleRebuilder {
     }
   }
 
+  /**
+   * Removes symlinks created by node-gyp for external dependencies, such as Python.
+   * We do not want to include these in rebuilt, packaged apps.
+   */
+  async removeNodeGypBins(): Promise<void> {
+    const buildLocation = path.resolve(this.modulePath, 'build');
+
+    d('checking existance of node_gyp_bins', buildLocation);
+    const nodeGypBinsLocation = path.resolve(buildLocation, 'node_gyp_bins');
+    const exists = await fs.pathExists(nodeGypBinsLocation);
+  
+    if (exists) {
```

Comment:
question: what are the downsides to putting the await in the conditional?

Reply:
Fair question - reading the fs docs, it looks like `fs.remove` will either remove the directory if it exists, or just silently continue, so I think the conditional here is actually unnecessary. I'll remove it 👍